pcsx2/gui/Dialogs/ConvertMemoryCardDialog.cpp

```diff
@@ -84,15 +84,15 @@ Dialogs::ConvertMemoryCardDialog::ConvertMemoryCardDialog( wxWindow* parent, con
 void Dialogs::ConvertMemoryCardDialog::CreateControls( const MemoryCardType sourceType ) {
 	m_text_filenameInput = new wxTextCtrl( this, wxID_ANY, wxEmptyString, wxDefaultPosition, wxDefaultSize, wxTE_PROCESS_ENTER );
 
-	RadioPanelItem toFile8MB = RadioPanelItem( _( "8MB File" ), pxE( L"Convert this memory card to a standard 8 MB Memory Card .ps2 file." ) )
+	RadioPanelItem toFile8MB = RadioPanelItem( _( "8 MB File [most compatible]" ), pxE( L"Convert this memory card to a standard 8 MB Memory Card .ps2 file." ) )
 		.SetInt( MemoryCardConversionType::MemoryCardConversion_File_8MB );
-	RadioPanelItem toFile16MB = RadioPanelItem( _( "16MB File" ), pxE( L"Convert this memory card to a 16 MB Memory Card .ps2 file." ) )
+	RadioPanelItem toFile16MB = RadioPanelItem( _( "16 MB File" ), pxE( L"Convert this memory card to a 16 MB Memory Card .ps2 file." ) )
 		.SetInt( MemoryCardConversionType::MemoryCardConversion_File_16MB );
-	RadioPanelItem toFile32MB = RadioPanelItem( _( "32MB File" ), pxE( L"Convert this memory card to a 32 MB Memory Card .ps2 file." ) )
+	RadioPanelItem toFile32MB = RadioPanelItem( _( "32 MB File" ), pxE( L"Convert this memory card to a 32 MB Memory Card .ps2 file." ) )
 		.SetInt( MemoryCardConversionType::MemoryCardConversion_File_32MB );
-	RadioPanelItem toFile64MB = RadioPanelItem( _( "64MB File" ), pxE( L"Convert this memory card to a 64 MB Memory Card .ps2 file." ) )
+	RadioPanelItem toFile64MB = RadioPanelItem( _( "64 MB File" ), pxE( L"Convert this memory card to a 64 MB Memory Card .ps2 file." ) )
 		.SetInt( MemoryCardConversionType::MemoryCardConversion_File_64MB );
-	RadioPanelItem toFolder = RadioPanelItem( _( "Folder" ), _( "Convert this memory card to a folder of individual saves." ) )
+	RadioPanelItem toFolder = RadioPanelItem( _( "Folder" ), _( "Convert this memory card to a folder of individual saves. Choose this to go beyond the save-limits of 8-64MB cards and if you only use PCSX2. You can always convert back." ) )
```

Comment:
This one too.

Reply:
unexpected end of file in macro expansion.
According to WX documentation: wxSTRINGIZE_T (  x  )  
I've tried different things but it gives those errors

Reply:
Try
```
	RadioPanelItem toFolder = RadioPanelItem( _( "Folder" ), _(
		"Convert this memory card to a folder of individual saves. "
		"Choose this to go beyond the save-limits of 8-64MB cards and if you only use PCSX2. "
		"You can always convert back." ) )
```

Reply:
Thanks, seems i was missing the space between ) )